modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -57,9 +57,29 @@ A standard {product-title} installation creates the following vCenter resources:
 ** 3 control plane nodes
 ** 3 compute machines
 
-Although these resources use 856 GB of storage, the bootstrap node is destroyed during the cluster installation process. A minimum of 800 GB of storage is required to use a standard cluster.
+These resources use 856 GB of storage in total.
 
-If you deploy more compute machines, the {product-title} cluster will use more storage.
+Bootstrap node resources are hard-coded and non-configurable. The bootstrap node uses the following resources:
```

Comment:
For user facing documentation just saying: "Bootstrap node resources are non-configurable." Is probably better. I would leave out "hard-coded"

Reply:
Thx! Updated.